src/groundcrew/utils.py

```diff
@@ -97,12 +97,15 @@ def build_llm_completion_client(
         completion = openaiapi.start_chat(model, client)
 
         def chat_complete(prompt):
-            messages = [
-                openaiapi.SystemMessage("You are a helpful assistant."),
-                openaiapi.UserMessage(prompt)
-            ]
-            response = completion(messages)
-            return response.content
+            try:
+                messages = [
+                    openaiapi.SystemMessage("You are a helpful assistant."),
+                    openaiapi.UserMessage(prompt)
+                ]
+                response = completion(messages)
+                return response.content
+            except:
```

Comment:
Bare exception.  Not a strong opinion in this situation, but `except Exception:` will placate the linter.

Reply:
Good catch!